BUILDGUIDE.md

```diff
@@ -123,6 +123,7 @@ Manual Tests require the below setup to run:
   |FileStreamDirectory | (Optional) If File Stream is enabled on SQL Server, pass local directory path to be used for setting up File Stream enabled database. |  `D:\\escaped\\absolute\\path\\to\\directory\\` |
   |UseManagedSNIOnWindows | (Optional) Enables testing with Managed SNI on Windows| `true` OR `false`|
   |IsAzureSynpase | (Optional) When set to 'true', test suite runs compatible tests for Azure Synapse/Parallel Data Warehouse. | `true` OR `false`|
+  |MakecertPath | The location of makecert.exe. This is not required if the path is present in the PATH environment variable. | Location of makecert.exe on machine |
```

Comment:
```suggestion
  |MakecertPath | The location of makecert.exe. This is not required if the path is present in the PATH environment variable. | Full path of makecert.exe on machine |
```

Reply:
I just noticed the value above for FileStreamDirectory : `D:\\escaped\\absolute\\path\\to\\directory\\`. Should we do something similar here? e.g.  `D:\\escaped\\absolute\\path\\to\\makecert.exe\\`

Reply:
Regarding your change at [CertificateUtilityWin](https://github.com/dotnet/SqlClient/pull/1271/files#diff-a0826e84a92307f80be30889258cc82faaa5fe0d009e6fe366dea0e8c38137b4R34) you're supposed to read a full path that includes `makecert.exe`. So, if it was correct the "Location" word couldn't clearly explain your purpose here!